src/app/components/common/node-editor/custom-notification-node/custom-notification-node.component.html

```diff
@@ -23,19 +23,21 @@
         </div>
       </div>
     </div>
-    <div>
-      <div class="is-flex is-justify-content-center">
+    <div class="columns is-centered">
+      <div class="column is-half">
+        <span>
+          <i class="is-custom-center bi bi-lightning-charge bi-lg no-hover" [ngClass]="notification.isDeliveryEnabled === 'true' ? 'has-text-info': 'has-text-grey-light'"></i>
+        </span>
+      </div>
+    </div>
+    <div class="columns mb-0 is-centered">
+      <div class="column is-half">
         <span class="is-hovered has-text-grey-light">
-          <p class="onhover-text">
-            <span class="help-text" [hidden]="!showPluginName">{{notification.channel}}</span>
-          </p>
-          <i class="bi bi-lightning-charge bi-lg no-hover pl-3" [ngClass]="notification.isDeliveryEnabled === 'true' ? 'has-text-info': 'has-text-grey-light'"></i>
-          <span class="help-text">{{notification.notificationType}}</span>
+          <span class="is-custom-center onhover-text help-text" [hidden]="!showPluginName">{{notification.channel}}</span>
```

Comment:
Do we need 2 classes? is-hovered and onhover-text?

Reply:
Yes
`is-hovered` is from style.css with property-
`cursor: pointer !important;`

While `onhover-text` is custom class with property-
`height: 0.9rem`